packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -725,6 +731,7 @@ export class Function extends FunctionBase {
     });
 
     this.runtime = props.runtime;
+    this.timeout = props.timeout;
```

Comment:
Why not reify this to the 3 seconds we promised it would be?

Reply:
@rix0rrr See previous discussion: https://github.com/aws/aws-cdk/pull/17000#discussion_r739116360